lib/sharezone_plus/sharezone_plus_page_ui/lib/src/call_to_action_button.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2024 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'package:flutter/material.dart';
+import 'package:sharezone_widgets/sharezone_widgets.dart';
+
+class CallToActionButton extends StatelessWidget {
```

Comment:
Should we move this to `sharezone_widgets` or keep it in `sharezone_plus_page_ui` for now?

Reply:
Not sure, just do what you think is the best

Reply:
I would keep it in `sharezone_plus_page_ui`. If we need anywhere else, we can move it to `sharezone_widgets`.